src/tests/tests/utils.js

```diff
@@ -467,4 +467,56 @@ QUnit.test("getHostFromDomainInput", assert => {
   );
 });
 
+// Tests algorithm used in the pixel tracking heuristic
+// It should return a common substring between two given values
+QUnit.test("findCommonSubstrings", assert => {
+
+  assert.equal(
+    utils.findCommonSubstrings('www.google.com', 'www.google.com/analytics/')[0].length >= 8,
+    true,
```

Comment:
Too bad there is no "assert greater than" in qUnit ... A bit more direct to use `assert.ok` here then, instead of comparing to `true`.

Reply:
But I'm not sure this assertion makes sense ... We know the exact length of the common substring for these two strings. But this doesn't prove that all matches have length greater or equal to 8 though.

Should we just test that the substring is exactly what we expect it to be?

Reply:
We could have another assertion that shows that a common substring under our length threshold is ignored.

Reply:
Added an assertion to test for substrings under the length threshold are ignored. For the one that was already there, do you think it's overkill to make a randomly generated string of common characters on the fly in this test?

Reply:
Randomness is not a good idea in unit tests generally because it might generate hard-to-reproduce failures.

Is there a code path you are trying to exercise here that isn't exercised by other tests?